src/Approximations/box_approximation.jl

```diff
@@ -73,6 +77,19 @@ function box_approximation(S::LazySet{N}) where {N}
     return Hyperrectangle(c, r)
 end
 
+function box_approximation(cap::Intersection{N,T1,T2}) where {N,T1,T2}
+    if ispolyhedral(cap.X) && ispolyhedral(cap.Y)
```

Comment:
I think this should be `||`. If any of the two are bounded, then the result is bounded.